specification/eventgrid/data-plane/Microsoft.ApiCenter/stable/2018-01-01/ApiCenter.json

```diff
@@ -0,0 +1,75 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Schema of Azure API Center service events published to Azure Event Grid",
+    "description": "Describes the schema of the Azure API Center service events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "ApiDefinitionAddedEventData": {
```

Comment:
We should prefix with ApiCenter for everything

Reply:
Yup, done.